.github/workflows/maven.yaml

```diff
@@ -227,17 +268,35 @@ jobs:
             TEST_ARG=$(echo ${sliced_array[@]} | sed 's/ /,/g')
 
             echo "will run tests : ${TEST_ARG[@]}"
-
-            ./mvnw -s .settings.xml \
-                -DtestsToRun=${TEST_ARG[@]} \
-                -e clean install \
-                -U -P sonar -nsu --batch-mode \
-                -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn \
-                -Dhttp.keepAlive=false \
-                -Dmaven.wagon.http.pool=false \
-                -Dmaven.wagon.http.retryHandler.class=standard \
-                -Dmaven.wagon.http.retryHandler.count=3 \
-                -Dskip.build.image=true
+          
+            if [[ $baseBranch == "2.1.x" ]]; then
+          
+              ./mvnw -s .settings.xml -pl '-:kubernetes-leader-election-example' -pl '-:kubernetes-hello-world-example' \
```

Comment:
skip examples and controllers, otherwise the pipeline runs > 1h, and it gets killed by github